src/alias/index.ts

```diff
@@ -0,0 +1,77 @@
+import { ConnectionREST } from '../index.js';
+import { WeaviateAlias, WeaviateAliasResponse } from '../openapi/types.js';
+import { Alias, CreateAliasInput, UpdateAliasInput } from './types.js';
+
+export interface Aliases {
+  /**
+   * Create alias for a collection.
+   *
+   * The collection must exist prior to aliasing it.
+   * One alias cannot be created for multiple collections simultaneously.
+   *
+   * @param {string} [opt.collection] Original collection name.
+   * @param {string} [opt.alias] Alias for collection.
+   * @returns {Promise<void>} Awaitable promise.
+   * */
+  create: (opt: CreateAliasInput) => Promise<void>;
+
+  /**
+   * List all aliases defined in the schema.
+   *
+   * @param {string | undefined} collection Get all aliases defined for this collection.
+   * @returns {Promise<Alias[] | undefined>} An array of aliases.
+   */
+  listAll: (collection?: string) => Promise<Alias[] | undefined>;
```

Comment:
Since `collection?` is optional here, what do you think about introducing a type, say `AliasListAllOpts` in case we ever need to extend this API to include other query options when listing aliases?